tests/yast2_gui/yast2_control_center.pm

```diff
@@ -270,6 +270,12 @@ sub start_printer {
 
 sub run {
     my $self = shift;
+    if (check_var('DISTRI', 'sle') && sle_version_at_least '15') {
```

Comment:
should be `is_sle && sle_version_at_least('15')` but actually I think we can just accept this as default behaviour because the following installation will not fail if the package is already there

Reply:
Ah, yeah , right. Thanks.